nilearn/plotting/tests/test_find_cuts.py

```diff
@@ -133,3 +137,11 @@ def test_find_cuts_empty_mask_no_crash():
     cut_coords = assert_warns(UserWarning, find_xyz_cut_coords, img,
                               mask=mask)
     np.testing.assert_array_equal(cut_coords, [.5, .5, .5])
+
+
+def test_fast_abs_percentile_no_index_error_find_cuts():
+    # check that find_cuts functions are safe
+    iris = load_iris()
+    X, _ = iris.data, iris.target
+    X, _ = to_niimgs(X, [2, 2, 2])
+    assert_equal(len(find_xyz_cut_coords(mean_img(X))), 3)
```

Comment:
I can reproduce a similar error with:

```python
import numpy as np

import nibabel

from nilearn.plotting.find_cuts import find_xyz_cut_coords

data = np.array([
    [[1., 2.],
     [3., 4.]],
    [[0., 0.],
     [0., 0.]]])

img = nibabel.Nifti1Image(data, np.eye(4))

find_xyz_cut_coords(img)

```

i.e. no need to resort to load_iris, to_niimgs and mean_img.